orc8r/cloud/helm/orc8r/charts/certs/values.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Create orchestrator certs.
+create: true
+
+dnsDomain: localhost
```

Comment:
minor nit: Let's just call this domain_name, calling it dns domain sounds redundant.

Reply:
Updated the name to `domainName`